classes/PrestaShopLogger.php

```diff
@@ -135,6 +147,16 @@ public static function addLog($message, $severity = 1, $errorCode = null, $objec
             $log->object_id = (int) $objectId;
         }
 
+        if (!empty($objectType)) {
+            $log->id_lang = (int) Context::getContext()->language->id;
+
+            if (Shop::getContext() == Shop::CONTEXT_SHOP) {
+                $log->id_shop = (int) Context::getContext()->shop->getContextualShopId();
+            } elseif (Shop::getContext() == Shop::CONTEXT_GROUP) {
```

Comment:
We couldn't have both `id_shop` and `id_shop_group`?

Reply:
Hi @PierreRambaud  Indeed, the basic idea is to know in which shop actions were applied on the object, if we have **`id_shop`**, **`id_shop_group`** will not really interest us. And if the action was done in a shop group context, it's **`id_shop_group`** we need, because **`id_shop`** will not have it anyway in this case. I totally agree that in the first case, we can also get the **`id_shop_group`** ... but to do what? (it's debatable)

Reply:
I think you're in better position than me to do suggestions :D
I'm waiting for Product Manager to take a decision. But the code looks good to me :)